api/OscillatorNode.json

```diff
@@ -201,10 +201,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/OscillatorNode/onended",
           "support": {
             "chrome": {
-              "version_added": "14"
+              "version_added": "30"
```

Comment:
Commented elsewhere, but can you also update `start` and `stop` for `OscillatorNode`?